README.md

```diff
@@ -72,24 +71,31 @@ Replace value of `image` field with your catalog source image.
 
 6. Deploy Che using chectl:
 ```sh
-chectl server:start --installer=olm --multiuser --platform=openshift -n che-namespace --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml --olm-channel=nightly --package-manifest-name=eclipse-che-preview-openshift
+chectl server:start --installer=olm --multiuser --platform=openshift \
+  -n che-namespace \
+  --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml \
+  --package-manifest-name=eclipse-che-preview-openshift \
+  --olm-channel=nightly 
 ```
 
 ### OpenShift oAuth
 
-Bear in mind that che-operator service account needs to have cluster admin privileges so that the operator can create oauthclient at a cluster scope.
-There is `oc adm` command in both deploy scripts. Uncomment it if you need this feature.
-Make sure your current user has cluster-admin privileges.
+Che-operator service account needs to have cluster admin privileges so that the operator can create a cluster-wide oauthclient. 
+
+Log in using a user with cluster admin privileges, then to enable this feature, run:
```

Comment:
Actually the working algorithm is:
1. login as cluster admin
2. create a user
3. log out from cluster admin
4. login as the user (no admin rights)
5. log out from user account
6. log in as admin
7. deploy Che with `--os-oauth` flag

Reply:
could that be simplified to:

1. login as cluster admin
2. create a user with admin rights
3. log out from cluster admin
4. log in as admin user 
5. deploy Che with --os-oauth flag

or:

1. login as cluster admin
2a. create a user with admin rights (for deployment only)
2b. create a user without admin rights (to use Che/CRW)
3. log out cluster admin
4. log in as admin user 
5. deploy Che with --os-oauth flag using admin user
6. log out admin user
7. open browser, log in as non-admin user

Reply:
The idea there is to have at least one valid user for OAuth (kubeadmin isn't such a user).
I think approach with admin user should work, but haven't checked it.